internal/security/svid/svid_test.go

```diff
@@ -0,0 +1,229 @@
+// Package cert provides functions to generate and load  certificates.
+package svid
+
+import (
+	"crypto/x509"
+
+	"net/url"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/h-fam/errdiff"
+)
+
+var (
+	caRSACERT = `-----BEGIN CERTIFICATE-----
```

Comment:
i'm not a fan of 'jam a ton of wierdo text' into test data, and prefer testdata files for this sort of thing. Is there a reason to NOT do this in that manner?

these probably also should be const not var ...

Reply:
changed it to const. I am also not the fan of wierdo text. However, the current CI check does not allow PEM files to be added as test data. I have opened an issue (https://github.com/openconfig/featureprofiles/issues/2098) but that did not get the answer, that is why changed the code to use the text.

Reply:
oh.. I wonder if this was why I didn't leave .pem files in the testing bits! :( ugh.  ok.

Reply:
it seems the check still is failing
```
Checking for allowed file types...
./internal/security/svid/testdata/ecdsa/ca-ecdsa-key.pem:                                                                                              PEM EC private key
Error: files should be in plain text or non-empty.
./internal/security/svid/testdata/rsa/ca-rsa-key.pem:                                                                                                  PEM RSA private key
Checking for regular files...
Checking for script files...
Please see "Allowed File Types" in CONTRIBUTING.md for detail.